packages/editor-ui/src/store.ts

```diff
@@ -652,9 +655,9 @@ export const store = new Vuex.Store({
 		},
 
 		setWorkflowPinData(state, pinData: IPinData) {
-			Vue.set(state.workflow, 'pinData', pinData);
+			Vue.set(state.workflow, 'pinData', pinData || {});
```

Comment:
Suggestion: `setWorkflowPinData(state, pinData: IPinData = {}) { ... }` instead of setting them below.

Reply:
On second thought, maybe `null` does not trigger the default arg.